frontend/src/pages/LoginPage.tsx

```
@@ -0,0 +1,23 @@
+import React from "react";
+
+interface LoginPageProps {
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)